src/views/game/ResultView.vue

```diff
@@ -18,11 +20,26 @@ const gameResultsDimensions = useElementBounding(gameResultsEl)
 const startingTransform = ref('')
 const displayTracks = ref(false)
 const displayButton = ref(false)
+const resultsAnimationPending = ref(true)
+const { width: screenWidth } = useWindowSize()
+const isMobileSize = computed(() => screenWidth.value < 1024)
```

Comment:
zwracasz boolean, wiec samo `isMobile` wystarczy

Reply:
1024 do jakis wspolnych constów

Sidenote, `<1024` to raczej sprawdzenie czy to tablet i w dół i raczej powinno występować w kodzie rozgraniczenie między:
- mobile
- tablet
- desktop